src/Trakx.Circle.ApiClient.Tests/Integration/CircleClientTestsBase.cs

```diff
@@ -34,7 +34,7 @@ public class CircleApiFixture : IDisposable
 
         public CircleApiFixture()
         {
-            var secrets = new Secrets();
+            var secrets = new Secrets(Config.API_KEY);
```

Comment:
As you upgrade all nuget packages from the solution later on, you will see that this secret has been deprecated and we use the `ConfigurationHelper` from `Trakx.Utils.Testing`